src/client/store/configureStore.ts

```diff
@@ -12,8 +12,8 @@ const createEnhancer = () => {
   const composeEnhancers =
     process.env.NODE_ENV !== 'production' &&
     typeof window === 'object' &&
-    window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
-      ? window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({})
+    (window as any).__REDUX_DEVTOOLS_EXTENSION_COMPOSE__
```

Comment:
I think this is necessary. see https://github.com/hiroppy/ssr-sample/blob/master/src/types/global.d.ts

Reply:
Yeah, you are right. I'll revert changes made to ts files.
cross-env should be merged though